### ISSUE TITLE
ender bag only working with ender chests

### ISSUE THREAD

**Issue by ToserBan** [maruohon/enderutilities#77]
Hi, I´m playing the modpack sevtech ages version 3.0.8(now in age 4). I tried to link the ender bag with a vanilla chest but the ender bag didnt turned purple and didnt let me open it. The only inventory that works is the vanilla ender chest, but thats useless for me, because its not possible to hook a pipe or a hopper to a vanilla ender chest. The modpack uses 1.12.2.0.7.8 of ender utilities. I googled around 2 hours, but nobody seems to have this problem.

**Comment by ToserBan** [comment]
Yes, that was my intention. Thanks for the hint, the item pickup manager works fine, even cross dimensional. Only its filters seem to not work or idk how they work, but thats no problem. I was just searching for a replacement for the ender pouch from the ender storage mod which is not part of sevtech ages.

**Comment by maruohon** [comment]
The IPM has two separate filters, one for the remote transport, and the other for allowing the player to pick up items from the ground. both filters can be individually enabled or disabled, and "bypass this filter" means that the filter won't be checked at all, and if I remember right, for the remote transport it means that the items will not be attempted to transport. They are checked in the order of "top to bottom" in the GUI, so basically items that are picked up from the ground, will first be checked against the transport filter (if it's enabled), and if it's allowed, the item will be transported to the linked inventory. If it wasn't enabled, or not allowed by the filter, then the bottom filter is checked next, to see if the player should be allowed to pick up that item. If that one also doesn't allow the item, then the player will not pick up that item from the ground at all, as long as the IPM is enabled and in the player's inventory.

**Comment by ToserBan** [comment]
Ah, i just have to check the "bypass this functionality" arrow. Now it works. Well great mod, thanks for your help. Can i use the mod in my next modpack ?

**Comment by maruohon** [comment]
Yes, of course. Wouldn't be much point in me making the mod if people wouldn't be allowed to use it.

**Comment by Keridos** [comment]
small suggestion: While you may not be reliably be able to reproduce the old behaviour, wouldn't it be rather easy to add in functionality to link the bad to ender utilities inventories (as in memory and handy chests). While restraining the usability by a bit it would suffice for the needs of most users probably.
